lib/stdlib/doc/src/queue.xml

```diff
@@ -421,6 +546,16 @@
         <p>Returns <c><anno>Item</anno></c> at the front of queue
           <c><anno>Q</anno></c>.</p>
         <p>Fails with reason <c>empty</c> if <c><anno>Q</anno></c> is empty.</p>
+          <p><em>Example 1:</em></p>
+          <pre>
+1> <input>Queue = queue:from_list([1,2,3,4,5]).</input>
+2> <input>1 == queue:get(Queue).</input>
+</pre>
+        <p><em>Example 2:</em></p>
```

Comment:
When the examples are very short (with no explaining text between them), I think it makes better sense to combine them. In my opinion, that is easier to read.